Added future plans and reworded for GA (#639)
timescale/docs.timescale.com-content: release-notes.md

## Future Plans

> TimescaleDB is an open source project with a vibrant community. 
> We are currently focusing on making our priorities known by that community and you are free to visit us on our Github repo and meet us on our [community Slack](https://timescaledb.slack.com).

Reviewer: ```suggestion
TimescaleDB is an open-source project with a vibrant community. 
We are currently focusing on making our priorities known by that community; 
we welcome you to visit our Github repo or join our [Slack community](https://slack.timescale.com).
```
```

### What to expect from our next release

> From our next release (2.0.1) we are planning to bring:

Reviewer: ```suggestion
For our next release (2.0.1), we plan to add:
```

> - [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 was released on September 24, 2020 and offers significant improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.

Reviewer: ```suggestion
- [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 offers various improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.
```

> - Backup and restore for multi-node deployments so our users can restore their data to a consistent state across all nodes.

Reviewer: ```suggestion
- Consistent distributed restore points for multi-node deployments, so our users can employ backup and restore across entire multi-node clusters, and ensure that restores can happen to a transactionally-consistent point-in-time.
```

> - Adding and renaming columns to compressed hypertables as a step towards advancing our support for compressed hypertables.

Reviewer: ```suggestion
- The ability to add and rename columns to compressed hypertables, as a step towards advancing our support for compressed hypertables (and enable them to be more mutable).
```

> - As well as some bug fixes.

Reviewer: ```suggestion
- Various bug fixes.
```

> From our next release (2.0.1) we are planning to bring:
> - [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 was released on September 24, 2020 and offers signifficant improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.
> - Backup and restore for multi-node deployments so our users can restore their data to a consisstent state across all nodes.
> - Adding and renaming columns to compressed hypertables as a step towards advancing our suport for compressed hypertables.
> - As well as some bug fixes.

Reviewer 2: If we announce it this way we are (indirectly) committing to deliver features in a specific release, which means that we have to leave the date open (for our release) in case we have delays. This contrasts with "we just deliver what we have at this date", which I think is fine. Users are more interested in what we plan for the next minor/major release rather than committing to a specific date.

I think we can remove the "as well as some bug fixes". It goes without saying.

```suggestion
From our next release (2.0.1) we are planning to bring:
- [Support for PostgreSQL 13](https://github.com/timescale/timescaledb/issues/2779): PG13 was released on September 24, 2020 and offers significant improvements in performance and usability. We want to provide our users the latest and greatest versions of PG.
- Backup and restore for multi-node deployments so our users can restore their data to a consistent state across all nodes.
- Adding and renaming columns to compressed hypertables as a step towards advancing our support for compressed hypertables.
- As well as some bug fixes.
```

> We are currently in the Generally Availble (GA) version 2.0.

Reviewer 2: ```suggestion
We are currently in the Generally Available (GA) version 2.0.
```

- **New and improved informational views**: Get more insight about hypertables,
  chunks, policies, and job scheduling.

> >:TIP: TimescaleDB 2.0 is currently GA, we encourage

Reviewer: Cut this Tip?  It's mainly about it being an RC, not GA.

Author: You are completely right.
